Upstream: Yes
Source: https://chromium.googlesource.com/chromium/src/+/cbdb8bd6567c8143dc8c1e5e86a21a8ea064eea4

From cbdb8bd6567c8143dc8c1e5e86a21a8ea064eea4 Mon Sep 17 00:00:00 2001
From: Maksim Sisov <msisov@igalia.com>
Date: Fri, 7 Sep 2018 18:57:42 +0000
Subject: [PATCH] OmniboxTextView: fix gcc error for structure initialization

It looks like there is bug in GCC 6, which cannot go through
structure initialization normally.

Thus, instead of a default initialization of one of the members,
explicitly initialize it to a default value.

Change-Id: Ia55cc6658e6b6b2f8a80c2582dd28f001c9e648c
Reviewed-on: https://chromium-review.googlesource.com/1213181
Reviewed-by: Scott Violet <sky@chromium.org>
Commit-Queue: Maksim Sisov <msisov@igalia.com>
Cr-Commit-Position: refs/heads/master@{#589614}
---
 chrome/browser/ui/views/omnibox/omnibox_text_view.cc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/chrome/browser/ui/views/omnibox/omnibox_text_view.cc b/chrome/browser/ui/views/omnibox/omnibox_text_view.cc
index f0a8083dc9300..9021284f166de 100644
--- chrome/browser/ui/views/omnibox/omnibox_text_view.cc
+++ chrome/browser/ui/views/omnibox/omnibox_text_view.cc
@@ -175,7 +175,8 @@ void ApplyTextStyleForType(SuggestionAnswer::TextStyle text_style,
       style = {part_color, .baseline = gfx::SUPERIOR};
       break;
     case SuggestionAnswer::TextStyle::BOLD:
-      style = {part_color, .weight = gfx::Font::Weight::BOLD};
+      style = {part_color, .baseline = gfx::NORMAL_BASELINE,
+               .weight = gfx::Font::Weight::BOLD};
       break;
     case SuggestionAnswer::TextStyle::NORMAL:
     case SuggestionAnswer::TextStyle::NORMAL_DIM:
